bin/init.bash

```diff
@@ -204,6 +202,8 @@ set_nginx_config() {
             use_proxy_protocol=false
             use_host_port=true
             service_enabled=false
+            yq4 --inplace '.networkPolicies.global.externalLoadBalancer = true' "${file}"
+            yq4 --inplace '.networkPolicies.global.ingressUsingHostNetwork = true' "${file}"
```

Comment:
Looks a bit out of place writing here.
Would it make sense to take the same approach as the other fields, aka using replace_set_me?

Reply:
[by default](https://github.com/elastisys/compliantkubernetes-apps/blob/main/config/config/common-config.yaml#L749-L750) this values are set to `false`, not sure if replace_set_me will help in this case

Reply:
done, please check